homeassistant/components/teslemetry/coordinator.py

```diff
@@ -205,8 +205,6 @@ async def _async_update_data(self) -> dict[str, Any]:
         except TeslaFleetError as e:
             raise UpdateFailed(e.message) from e
 
-        self.updated_once = True
```

Comment:
This should be unrelated to this PR. I suppose maybe you change the wrong file as this is the `teslemetry` integration?

Reply:
Ooops